src/main/java/uk/gov/hmcts/juror/job/execution/jobs/dashboard/ams/data/BureauLettersAutomaticallyGenerated.java

```diff
@@ -1,62 +1,76 @@
 package uk.gov.hmcts.juror.job.execution.jobs.dashboard.ams.data;
 
 import lombok.extern.slf4j.Slf4j;
-import uk.gov.hmcts.juror.job.execution.client.contracts.SchedulerServiceClient;
+import uk.gov.hmcts.juror.job.execution.config.DatabaseConfig;
 import uk.gov.hmcts.juror.job.execution.jobs.Job;
-import uk.gov.hmcts.juror.job.execution.jobs.letter.LetterJob;
+import uk.gov.hmcts.juror.job.execution.service.contracts.DatabaseService;
 
+import java.time.Clock;
 import java.time.LocalDateTime;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
 
 @Slf4j
 @SuppressWarnings("PMD.LawOfDemeter")
 public class BureauLettersAutomaticallyGenerated extends DashboardDataEntry {
-    final SchedulerServiceClient schedulerServiceClient;
+    static final String BUREAU_AUTO_GEN_LETTERS_SQL = """
+    select
+        coalesce(sum(case when jh.history_code = 'RDIS' then 1 else 0 end),0) withdrawal,
+        coalesce(sum(case when jh.history_code = 'RRES' then 1 else 0 end),0) confirmation
+    from juror_mod.juror_history jh
+    where jh.user_id = 'AUTO' and jh.date_created = current_date;
```

Comment:
Just want to confirm current_date does not include a time element?
and date_created on the database also does not include a time element?

Reply:
Updated to cast date_created to date (without time)